src/main/java/az/ingress/model/request/CreateOrUpdateCommentRequest.java

```diff
@@ -0,0 +1,25 @@
+package az.ingress.model.request;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class CreateOrUpdateCommentRequest {
+
+    @NotNull(message = "User ID cannot be null")
```

Comment:
[Optional]:
1. move these messages to `ValidationMessageConstants` 
2. you can use i18n for exceptions